### PR TITLE
Added all promise calls and fixed sample data

### PR DESCRIPTION
**Pull request by Massacre-Saint**

### CODE REVIEW

**Review by gitNitroTitan — APPROVED** [review]

File: src/scripts/components/pages/viewOrders.js [hunk 1]
```diff
@@ -5,17 +5,18 @@ const viewOrders = (array) => {
...
-    <a href="#" class="card-link">Details</a>
-    <a href="#" class="card-link">Edit</a>
-    <a href="#" class="card-link">Delete</a>
-    </div>
-</div>`;
+    domString += `
+    <div class="order-card">
+      <div class="card-body">
+        <h5 class="card-title">${obj.orderName}</h5>
+        <h6 class="card-subtitle mb-2 text-muted">${obj.orderStatus}</h6>
```
Comment (lesleyjkeyes) [inline src/scripts/components/pages/viewOrders.js:12]: should be updated to closedStatus

File: src/scripts/api/itemData.js [hunk 2]
```diff
@@ -15,11 +15,40 @@ const getItems = () => new Promise((resolve, reject) => {
...
     })
     .catch((error) => reject(error));
 });
-export { getItems, deleteSingleItem };
+
+const createItem = (itemObj) => new Promise((resolve, reject) => {
+  axios.post(`${dbUrl}/items.json`, itemObj)
+    .then((response) => {
+      const payload = { firebaseKey: response.data.name };
+      axios.patch(`${dbUrl}/orders/${response.data.name}.json`, payload)
```
Comment (nishayaraj) [inline src/scripts/api/itemData.js:36]: items/ instead of /orders in the url

**Review by lesleyjkeyes — CHANGES_REQUESTED** [review]
see comments

**Review by lesleyjkeyes — APPROVED** [review]
looks good!

**Pull request merged**